libs/island-ui/core/src/lib/Filter/FilterInput/FilterInput.tsx

```diff
@@ -1,27 +1,31 @@
 import React from 'react'
 import { ResponsiveProp } from '../../../utils/responsiveProp'
-import { InputBackgroundColor } from '../../Input/types'
+import { InputBackgroundColor, InputIcon, InputButton } from '../../Input/types'
 import { Input } from '../../Input/Input'
 
 export interface FilterInputProps {
   name: string
   id?: string
   placeholder?: string
   value: string
+  button?: Omit<InputButton, 'name' | 'type'>
   onChange: (value: string) => void
   onKeyDown?: (
     event: React.KeyboardEvent<HTMLInputElement | HTMLTextAreaElement>,
   ) => void
   backgroundColor?: ResponsiveProp<InputBackgroundColor>
 }
 
+const icon: InputIcon = { name: 'search', type: 'outline' }
```

Comment:
I'm just wondering if this is in Figma, nice job adding a story to storybook for this 👍

Reply:
Good question. I can talk with design and make sure it's added.